src/discretized-collision-checking.cc

```diff
@@ -37,6 +38,7 @@ namespace hpp {
       DiscretizedPathValidation (robot, stepSize)
     {
       add (CollisionValidationPtr_t (CollisionValidation::create (robot)));
+      add (JointBoundValidationPtr_t (JointBoundValidation::create (robot)));
```

Comment:
For reference, I think these two lines should be swapped as joint bound validation is much faster to execute.

Reply:
Done in 48792c4